adafruit_magtag/magtag.py

```diff
@@ -65,16 +70,16 @@ class MagTag(PortalBase):
     def __init__(
         self,
         *,
-        url=None,
-        headers=None,
-        json_path=None,
-        regexp_path=None,
-        default_bg=0xFFFFFF,
-        status_neopixel=None,
-        json_transform=None,
-        rotation=270,
-        debug=False,
-    ):
+        url: Optional[str] = None,
+        headers: Optional[Dict] = None,
+        json_path: Optional[Sequence] = None,
```

Comment:
`Sequence` is also a generic so it should give the type of what it's a sequence of.  I believe it's `str` right?  So then it should be:

`Optional[Sequence[str]]`

Reply:
The docstring for `json_path` mentions it could be a list or a list of lists, so would it be as follows?
`json_path: Optional[Sequence[Union[Sequence[str], str]]] = None`

Here's that part of the docstring:
```
:param json_path: The list of json traversal to get data out of. Can be list of lists for
                      multiple data points. Defaults to ``None`` to not use json.
```

Reply:
Looks like the magtag_quote_demo.py example in the module passes a tuple with two lists in it, each list containing an int and a string. mypy isn't happy with `Optional[Sequence[str]]`. 

Is it reasonable to do this?
`json_path: Optional[Sequence[Union[Sequence[object], object]]] = None,`

Thanks!

Reply:
You could use `Any` perhaps, so maybe `Optional[Sequence[Union[Sequence[Any], Any]]]`?  @FoamyGuy what do you think?

Reply:
`Optional[Sequence[Union[Sequence[Any], Any]]]` looks good to me. Or even one less layer of detail `Optional[Sequence[Any]]` I think is okay as well. 

This one is a tricky one to type because the way the API is written it's very versatile and dynamic with what could be passed in this argument.

Reply:
Thanks! I put in Optional[Sequence[Any]] to preserve the versatility as @FoamyGuy mentioned.